KTXContainerFormat.md

```diff
@@ -0,0 +1,17 @@
+# KTX Container Format
+
+When implementing support for compressed textures, developers should be aware of three conceptual formats:
+
+1. **Transmission format:** A highly compressed representation of pixel data, in a layout designed for very efficient transcoding to one or more GPU compression formats.
+	- _Examples: ETC1S and UASTC_
+2. **GPU compressed pixel format:** A compressed representation of pixel data, understood by the GPU.
+	- _Examples: BCn, ASTC, ETC, etc._
+3. **Container format:** An explanatory wrapper around the transmission format data. Describes image dimensions, compression types, and how to access and transcode that data into a GPU-compatible pixel format. Without a container format, compressed data cannot be correctly moved between software applications.
+	- _Examples: KTX 2.0_
+
+ To represent textures within portable [glTF 2.0](https://github.com/KhronosGroup/glTF) 3D models, we use the [KTX 2.0 Container Format](http://github.khronos.org/KTX-Specification/)<sup>1</sup> (`.ktx2`). KTX 2.0 is a relatively simple binary data format, and can be read or written without the use of existing software, by referencing the format specification. Several existing implementations are available:
```

Comment:
The first sentence seems to be a bit misleading: glTF 2.0 core doesn't support KTX 2.0 textures; with the BasisU extension, glTF uses KTX container only for compressed textures.

How about something along these lines:
> Portable [glTF 2.0](https://github.com/KhronosGroup/glTF) 3D assets may use compressed textures stored in the [KTX 2.0 Container Format](http://github.khronos.org/KTX-Specification/)<sup>1</sup> (`.ktx2`) via `KHR_texture_basisu` extension.